osu.Game/OsuGame.cs

```diff
@@ -90,7 +90,7 @@ public class OsuGame : OsuGameBase, IKeyBindingHandler<GlobalAction>
 
         protected BackButton BackButton;
 
-        protected SettingsPanel Settings;
+        protected SettingsOverlay Settings;
```

Comment:
How is this even related to this PR? Should not be here.

Reply:
It's a corollary of the `Cache` -> `CacheAs` change [as I outlined here](https://github.com/ppy/osu/pull/8965#discussion_r421827375). I too dislike the subtlety of this.